<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainSlideWindow</class>
 <widget class="QMainWindow" name="MainSlideWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>procon2014-qt-slide</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="slideView">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>496</x>
      <y>140</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="shuffleButton">
       <property name="text">
        <string>Shuffle</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="shuffle2Button">
       <property name="text">
        <string>Shuffle!</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="reloadButton">
       <property name="text">
        <string>Reload</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>220</y>
      <width>191</width>
      <height>97</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Swapping Cost</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="swappingCostEdit">
       <property name="text">
        <string>10</string>
       </property>
       <property name="placeholderText">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLineEdit" name="selectionCostEdit">
         <property name="text">
          <string>20</string>
         </property>
         <property name="placeholderText">
          <string notr="true"/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="selectionLimitEdit">
       <property name="text">
        <string>5</string>
       </property>
       <property name="placeholderText">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Selection Cost</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Selection Limit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>40</y>
      <width>271</width>
      <height>94</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Size(2~16): </string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="HsizeSpinBox">
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="maximum">
        <number>16</number>
       </property>
       <property name="value">
        <number>3</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Animation Time:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QSpinBox" name="WsizeSpinBox">
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="maximum">
        <number>16</number>
       </property>
       <property name="value">
        <number>3</number>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QSpinBox" name="animationTimeSpinBox">
       <property name="maximum">
        <number>10000</number>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
       <property name="value">
        <number>300</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QScrollBar" name="animationTimeScrollBar">
       <property name="maximum">
        <number>1500</number>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
       <property name="value">
        <number>300</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="changeSelectedBlockButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>180</y>
      <width>271</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Change Selected Block</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="slideInfoBrowser">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>330</y>
      <width>256</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>540</y>
      <width>236</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="solveButton">
       <property name="text">
        <string>Solve</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="restoreButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Restore</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="playButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Play</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>450</y>
      <width>425</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Problem:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="problemComboBox">
       <item>
        <property name="text">
         <string>01 マグロ丼</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>02 とんこつラーメン</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>03 開発者</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>04 TATAMI</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>05 kyuridenamida</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>06 phycolow</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>07 開発者2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>08 OCHIBA</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>09 草原</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10 レナねいさん</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>11 レナねいさん2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>12 ポスター</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>13 沖縄高専</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>14 開発者3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>15 みじゅごろう</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>16 レンガ</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>17 開発者</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>01 データセット1 (P=48x36)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>02 データセット2 (P=48x36)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>03 データセット3 (P=48x36)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>04 データセット4 (P=48x36)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>05 データセット5 (P=48x36)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>06 データセット6 (P=48x36)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>07 データセット7 (P=48x36)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>08 データセット8 (P=48x36)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>09 データセット9 (P=48x36)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10 データセット10 (P=48x36)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>11 データセット11 (P=48x36)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>12 データセット12 (P=48x36)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>13 データセット13 (P=48x36)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>14 データセット14 (P=48x36)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>15 データセット15 (P=48x36)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>16 データセット16 (P=48x36)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>17 データセット17 (P=48x36)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>18 データセット18 (P=48x36)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>19 データセット19 (P=48x36)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>20 データセット20 (P=48x36)</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="readProblemButton">
       <property name="text">
        <string>Read</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>500</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Solver:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="solverComboBox">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>500</y>
      <width>151</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1. Straight Solver</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2. Exact Solver</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3. Hitode Solver</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4. Kurage Solver</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5. Dolphin Solver</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6. Dragon Solver</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7. Lizard Solver</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8. L2 Solver</string>
     </property>
    </item>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>560</y>
      <width>235</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Color:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="colorDistanceButton">
       <property name="text">
        <string>Distance</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="colorAccordanceButton">
       <property name="text">
        <string>Accordance</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>490</y>
      <width>431</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="1" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>POST:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>GET:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="postServerAddrEdit">
       <property name="text">
        <string>127.0.0.1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="postServerPortEdit">
       <property name="text">
        <string>22346</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="problemServerPortEdit">
       <property name="text">
        <string>22345</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="problemServerAddrEdit">
       <property name="text">
        <string>127.0.0.1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="startPostWorkerButton">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QPushButton" name="getCancelButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="postAnswerButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>POST</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QPushButton" name="postCancelButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="readProblemFromServerButton">
       <property name="text">
        <string>GET</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QCheckBox" name="autoPostCheckBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>560</y>
      <width>87</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Auto POST</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="autoSolveCheckBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>560</y>
      <width>87</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Auto Solve</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>ファイル(&amp;F)</string>
    </property>
    <addaction name="fileOpen"/>
   </widget>
   <addaction name="menu_F"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="fileOpen">
   <property name="text">
    <string>開く(&amp;O)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
